.github/workflows/run-forge.yaml

```diff
@@ -16,6 +16,11 @@ on:
         required: true
         type: string
         description: The Forge k8s namespace to be used for test. This value should manage Forge test concurrency. It may be truncated.
+      FORGE_TEST_SUITE:
```

Comment:
probably need to set `FORGE_TEST_SUITE` in `build-images.yaml` as well since that's where the land-blocking Forge is invoked